src/automated-testing/visual.fe.spec.ts

```diff
@@ -27,13 +27,24 @@ visualStories.forEach((story) => {
 
     const head = await page.locator('head #addon-backgrounds-color');
     const body = page.locator('body');
+    const img = page.locator('img');
 
     await expect(body)
       .not.toHaveClass('sb-show-preparing-story');
 
     await expect(head)
       .toBeAttached();
 
+    // Check if image exists on page
+    if (await img.count() > 0) {
+      await img.scrollIntoViewIfNeeded();
```

Comment:
we should not do this. if i'd like to test a page with an image, i don't want to scroll to the image necessarily. if you absolutely need this statement for the isolated image tests, then i would create a separate `beforeEach` helper, similar to `src/test-helpers/payload-login.ts` and include this beforeEach hoook in each test you would like to have it.